model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV2.java

```diff
@@ -0,0 +1,410 @@
+                                                            NDArrays.add(
+                                                                    list.get(0).singletonOrThrow(),
+                                                                    list.get(1)
+                                                                            .singletonOrThrow())),
+                                            Arrays.asList(block, Blocks.identityBlock())));
+        }
+        return block; // No reluBlock here
+    }
+
+    /**
+     * Builds a {@link Block} that represent multiple repeats of an inverted residual Unit.
+     *
+     * @param repeat the repeatTimes of an inverted residual Block
+     * @param inputChannels number of inputChannels of the block
+     * @param outputChannels number of outputChannels of the block
+     * @param stride the stride of an inverted residual Unit
+     * @param t the multipleTime of a pointWise Kernel
+     * @param batchNormMomentum the momentum of batchNormLayer
+     * @return a {@link Block} that represent several repeated inverted residual Units
+     */
+    public static Block makeStage(
+            int repeat,
+            int inputChannels,
+            int outputChannels,
+            int stride,
+            int t,
+            float batchNormMomentum) {
+        SequentialBlock layers = new SequentialBlock();
+        layers.add(linearBottleNeck(inputChannels, outputChannels, stride, t, batchNormMomentum));
+        while (repeat - 1 > 0) {
+            layers.add(linearBottleNeck(outputChannels, outputChannels, 1, t, batchNormMomentum));
+            repeat--;
+        }
+        return layers;
+    }
+
+    /**
+     * Creates a new {@link Block} of {@link MobileNetV2} with the arguments from the given {@link
+     * Builder}.
+     *
+     * @param builder the {@link Builder} with the necessary arguments
+     * @return a {@link Block} that represents the required MobileNetV2 model
+     */
+    public static Block mobilenetV2(Builder builder) {
+        SequentialBlock mobileNet = new SequentialBlock();
+        SequentialBlock pre = new SequentialBlock();
+        for (int i = 0; i < builder.repeatTimes[0]; i++) { // add as a sequence
+            pre.add(
+                            Conv2d.builder()
+                                    .setKernelShape(new Shape(1, 1))
+                                    .setFilters(builder.filters[0])
+                                    .optStride(new Shape(builder.strides[0], builder.strides[0]))
+                                    .optBias(false)
+                                    .build())
+                    .add(
+                            BatchNorm.builder()
+                                    .optEpsilon(2E-5f)
+                                    .optMomentum(builder.batchNormMomentum)
+                                    .build())
+                    .add(Activation.reluBlock());
+        }
+
+        Block bottleNeck1 =
```

Comment:
Rather than constructing each bottleNeck individually, can you create them by a for loop? Then, you can store them in an array and use `SequentialBlock.addAll()` to add all of them to `mobilenet` at once

Reply:
yeah,I realize that